docs/source/tutorials/memory_optimizations.rst

```diff
@@ -390,16 +400,16 @@ Quantized Low Rank Adaptation (QLoRA)
 
 *What's going on here?*
 
-`QLoRA <https://arxiv.org/abs/2305.14314>`_ is an enhancement on top of `LoRA <https://arxiv.org/abs/2106.09685>`_
+`QLoRA <https://arxiv.org/abs/2305.14314>`_ is a memory enhancement on top of `LoRA <https://arxiv.org/abs/2106.09685>`_
 that maintains the frozen model parameters from LoRA in 4-bit quantized precision, thereby reducing memory usage.
 This is enabled through a novel  4-bit NormalFloat (NF4) data type proposed by the authors, which allows for 4-8x less
 parameter memory usage whilst retaining model accuracy. You can read our tutorial on :ref:`finetuning Llama2 with QLoRA<qlora_finetune_label>`
 for a deeper understanding of how it works.
 
-When considering using QLoRA to reduce memory usage, it's worth noting that QLoRA prevents accuracy degradation during quantization
-by up-casting quantized parameters to the original higher precision datatype during model forward passes - this up-casting may
-incur penalties to training speed. The :ref:`relevant section <qlora_compile_label>` in our QLoRA tutorial demonstrates the usage of ``torch.compile``
-to address this by speeding up training.
+When considering using QLoRA to reduce memory usage, it's worth noting that a) QLoRA is slower than LoRA and may not be worth it if
+the model you are finetuning is small; b) QLoRA prevents accuracy degradation during quantization by up-casting quantized parameters
```

Comment:
I think this inline enumeration will not look great in the live docs. Also you can probably be more explicit about the memory savings, right? Doesn't it save roughly a constant 1.5 bytes * (# of model parameters) compared to LoRA? In general would try to be less handwavy about memory/perf statements in this doc wherever possible

Reply:
removed the a/b and added 1.5 bytes